themes/bootprint3/theme.config.php

```diff
@@ -1,4 +1,158 @@
+            'checked-out' => 'Fugue:book.png',
+            'cite' => 'Fugue:asterisk_orange.png',
+            'collapse-close' => 'Fugue:bullet_arrow_up.png',
+            'collapse-open' => 'Fugue:bullet_arrow_down.png',
+            'cover-replacement' => 'Fugue:images.png',
+            'delete' => 'Fugue:bin.png',
+            // 'dot' => inherited from FontAwesome
+            // 'dropdown-caret' => inherited from FontAwesome
+            'edit' => 'Fugue:edit.png',
+            'email' => 'Fugue:email.png',
+            'export' => 'Fugue:link_go.png',
+            'external-link' => 'Fugue:link_go.png',
+            'external-link-rtl' => 'Fugue:link_go_rtl.png',
+            'facet-checked' => 'Fugue:tick.png',
+            'facet-exclude' => 'Fugue:cross.png',
+            'facet-unchecked' => 'Fugue:tick_faded.png',
+            'favorites' => 'Fugue:star.png',
+            'feedback' => 'Fugue:email.png',
+            'hold' => 'Fugue:flag_red.png',
+            'hold-rtl' => 'Fugue:flag_red_rtl.png',
+            'ill-request' => 'Fugue:arrow_switch.png',
+            'loan-history' => 'Fugue:clock.png',
+            // 'menu-bars' => inherited from FontAwesome
+            'more' => 'Fugue:arrow_right.png',
+            'more-rtl' => 'Fugue:arrow_left.png',
+            'my-account' => 'Fugue:user.png',
+            'notification' => 'Fugue:bell.png',
+            'options' => 'Fugue:cog.png',
+            'overdrive' => 'Fugue:drive_web.png',
+            'page-first' => 'Fugue:resultset_first.png',
+            'page-first-rtl' => 'Fugue:resultset_last.png',
+            'page-last' => 'Fugue:resultset_last.png',
+            'page-last-rtl' => 'Fugue:resultset_first.png',
+            'page-next' => 'Fugue:resultset_next.png',
+            'page-next-rtl' => 'Fugue:resultset_previous.png',
+            'page-prev' => 'Fugue:resultset_previous.png',
+            'page-prev-rtl' => 'Fugue:resultset_next.png',
+            'print' => 'Fugue:printer.png',
+            'profile' => 'Fugue:user.png',
+            'qrcode' => 'Fugue:qrcode.png',
+            'recall' => 'Fugue:flag_red.png',
+            'recall-rtl' => 'Fugue:flag_red_rtl.png',
+            'remove' => 'Fugue:cross.png',
+            'return' => 'Fugue:arrow_right.png',
+            'return-rtl' => 'Fugue:arrow_left.png',
+            'rss' => 'Fugue:rss.png',
+            'save' => 'Fugue:disk.png',
+            'search' => 'Fugue:magnifier.png',
+            'sign-in' => 'Fugue:door_in.png',
+            'sign-out' => 'Fugue:door_out.png',
+            'sms' => 'Fugue:phone.png',
+            // 'spinner' => inherited from FontAwesome
+            'status-pending' => 'Fugue:time.png',
+            'status-ready' => 'Fugue:bell.png',
+            'status-retrieval' => 'Fugue:lorry.png',
+            'storage-retrieval' => 'Fugue:lorry.png',
+            'tree-context' => 'Fugue:treeCurrent.png',
+            'ui-add' => 'Fugue:add.png',
```

Comment:
Why ui- prefix with these two? e.g. remove is without a prefix.

Reply:
I hadn't updated bootprint3 yet, waiting for feedback on the new names.